modules/masterbar/masterbar.php

```diff
@@ -1375,18 +1341,16 @@ private function add_my_home_submenu_item( &$wp_admin_bar ) {
 			return;
 		}
 
-		if ( $this->is_my_home_enabled() ) {
-			$wp_admin_bar->add_menu(
-				array(
-					'parent' => 'blog',
-					'id'     => 'my-home',
-					'title'  => __( 'My Home', 'jetpack' ),
-					'href'   => 'https://wordpress.com/home/' . esc_attr( $this->primary_site_slug ),
-					'meta'   => array(
-						'class' => 'mb-icon',
-					),
-				)
-			);
-		}
+		$wp_admin_bar->add_menu(
+			array(
+				'parent' => 'blog',
+				'id'     => 'my-home',
+				'title'  => __( 'My Home', 'jetpack' ),
+				'href'   => 'https://wordpress.com/home/' . esc_attr( $this->primary_site_slug ),
```

Comment:
This is a little odd; I'd expect:

```suggestion
				'href'   => esc_url( "https://wordpress.com/home/$this->primary_site_slug" ),
```

Reply:
I think what Miguel has is correct. 

Examples in the following are a plain string: https://developer.wordpress.org/reference/classes/wp_admin_bar/add_node/

Reply:
Yeah, I just used what's being used [everywhere](https://github.com/Automattic/jetpack/blob/ca293860bb8ed90688c0ada2e0365f6979062cc1/modules/masterbar/masterbar.php#L879) [else](https://github.com/Automattic/jetpack/blob/ca293860bb8ed90688c0ada2e0365f6979062cc1/modules/masterbar/masterbar.php#L1027).

Reply:
>  I just used what's being used everywhere else

It _is_ used everywhere else, and I'd argue everywhere else is also not the best practice 🙃 But ultimately, this case is the same result 👍